### ISSUE TITLE
Screen Reader: Unclear Dismiss Action

### ISSUE THREAD

**Issue by dominicmh** [woltapp/wolt_modal_sheet#396]
## Bug report

If a sheet can be dismissed via drag (`showDragHandle = true`), it provides a corresponding semantic node with the label "Dismiss". 
The label should communicate to users that a swipe gesture is expected. Usually, users would expect a tap action. However if performing the tap action, nothing happens.

### Expected behavior

Either – and preferably – the label should be "Swipe down to dismiss" (double-finger swipe gesture in screen reader mode). Or the semantic node should react to [onTap](https://api.flutter.dev/flutter/semantics/SemanticsProperties/onTap.html) (double tap in screen reader moder).

### Steps to reproduce

1. Use the `WoltModalSheet` with the drag handle:
```dart
WoltModalSheet(
  …
  showDragHandle: true,
)
```
2. Use a screen reader or another tool to inspect the semantic layout. Activate the drag handle's semantic node. Example from Apple's Accessibility Inspector:

<img width="1066" height="639" alt="Image" src="https://github.com/user-attachments/assets/1d2e5848-a748-49fa-a2c1-b1120258537e" />

---

## Additional context

The relevant [Semantics](https://api.flutter.dev/flutter/widgets/Semantics-class.html) node is created in [WoltBottomSheetDragHandle](https://github.com/woltapp/wolt_modal_sheet/blob/06800fde072c3a7c1c8e19388b3b2daebe22cbbd/lib/src/widgets/wolt_bottom_sheet_drag_handle.dart#L19)

### Optional consideration

It would be nice to have an option to disable the semantic node altogether.
In our case (see screenshot), we have an additional button to dismiss the sheet because not all users are familiar with the drag to dismiss gesture. Having two semantic nodes for the same action, however, is not desirable and the explicit button would better suit in this case.
This is a low-prio feature request and not directly related to the bug report.

---

**Comment by dominicmh** [comment]
Some feedback from a blind tester in our accessibility tests:
- The view behind the modal should not have a semantic node (label "Modal barrier")
- It's unusual to have a semantic node for the bottom sheet itself (label "Bottom Sheet")

**Comment by shaheer-fusion** [comment]
facing the same issue can we get an ETA on this?
